arctic/chunkstore/_chunker.py

```diff
@@ -60,3 +60,13 @@ def exclude(self, data, range_obj):
         data, filtered by range_obj
         """
         raise NotImplementedError
+
+    def chunk_to_str(self, chunk_id):
+        """
+        Converts a chunk identifier to a string
```

Comment:
Maybe beef this up a bit.  What's a `chunk identifier` and what's the intent of the result?

Seems to only be used in `_checksum` calculation currently:

```
        sha.update(self.chunker.chunk_to_str(doc['start']).encode('ascii'))
        sha.update(self.chunker.chunk_to_str(doc['end']).encode('ascii'))
```

taking a date. Generally I would think `chunk identifier` would contain the `symbol` too or be the `sha`.